src/android/src/main/java/org/pgsqlite/SQLitePlugin.java

```diff
@@ -357,6 +357,7 @@ private void startDatabase(String dbname, JSONObject options, CallbackContext cb
     private SQLiteDatabase openDatabase(String dbname, String assetFilePath, int openFlags, CallbackContext cbc) throws Exception {
```

Comment:
Essentially you can choose whether you want to use the native or pure Java Android implementation. The native piece naturally has some pre-compiled native libraries to go along with the package. Nothing happens dynamically - you just choose whether you want to go with native or pure Java.. :)